### ISSUE TITLE
Cannot instal MSFSPluginPackage-3.0.1.lplug4- Exception: '...not found. An assembly manifest was expected in the module

### ISSUE THREAD

**Issue by hapandre** [calibx/msfsdeck#248]
I cannot install the 3.0 plugin. 

Calling LoupedeckPluginInstaller.exe ends with this error message:

PS C:\Program Files (x86)\Loupedeck\Loupedeck2> .\LoupedeckPluginInstaller ci MSFSPluginPackage-3.0.1.lp4
Exception: 'The file or assembly “file:///C:\Program Files (x86)\Loupedeck\Loupedeck2\MSFSPluginPackage-3.0.1.lp4” or a dependency of it was not found. **An assembly manifest was expected in the module.**

What can I do?

**Comment by calibx** [comment]
Did you try double clicking on the file ?

Which version of Loupedeck UI are you using ?
Can you open the lp4 with 7Zip ?

I will double check the release but if  I remember well I already have made a successfull installation test before releasing it.

**Comment by hapandre** [comment]
Yes I tried to start bij double-clicking first, but nothing happend. 
Only a cmd-window was showen for a short time but nothing displayed.

The version of de Loupedeck UI is 5.9.1.19364

I can open the file with zip en it seems to be okay. 
I then tried to install bij run the installer manually. 
This programm ends with this error-message:

PS C:\Program Files (x86)\Loupedeck\Loupedeck2> .\LoupedeckPluginInstaller ci MSFSPluginPackage-3.0.1.lp4
Exception: 'The file or assembly “file:///C:\Program Files (x86)\Loupedeck\Loupedeck2\MSFSPluginPackage-3.0.1.lp4” or a dependency of it was not found. An assembly manifest was expected in the module.

Is there a other posibility to install the driver manually?

**Comment by hapandre** [comment]
Is this project stil active???
